scss/elements/tables.scss

```diff
@@ -16,14 +10,57 @@
 
     tr {
       th + th {
-        box-shadow: -4px 0 $base-color;
+        box-shadow: -4px 0 $color;
       }
       td {
-        box-shadow: 0 -4px $base-color;
+        box-shadow: 0 -4px $color;
       }
       td:not(:first-child) {
-        box-shadow: -4px 0 $base-color, 0 -4px $base-color;
+        box-shadow: -4px 0 $color, 0 -4px $color;
       }
     }
   }
+  &.is-centered th {
+    text-align: center;
+  }
+  &.is-bordered {
+    box-shadow: 4px 0 $base-color, 0 -4px $base-color, -4px 0 $base-color, 0 4px $base-color;
+    @include thsAndTdsBoxShadow($base-color);
+  }
+  &.is-dark {
+    position: relative;
+    color: $background-color;
+    background-color: $base-color;
+    box-shadow: 4px -4px $base-color, -4px -4px $base-color, -4px 4px $base-color,
+      4px 4px $base-color;
+
+    &::before,
+    &::after {
+      position: absolute;
+      content: "";
+    }
+
+    &::after {
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      z-index: 1;
+    }
+
+    &::before {
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      background: linear-gradient(to left, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to bottom, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to right, transparent 4px, #fff 4px) no-repeat,
+        linear-gradient(to top, transparent 4px, #fff 4px) no-repeat;
```

Comment:
same here 🙇 
`$background-color`

Reply:
Changed all `#fff` to `$background-color`.